web/backend/src/Server.ts

```diff
@@ -7,8 +7,19 @@ import kyber from '@dedis/kyber';
 import crypto from 'crypto';
 import lmdb, { RangeOptions } from 'lmdb';
 import xss from 'xss';
+import createMemoryStore from 'memorystore';
 
-const config = require('../config.json');
+const MemoryStore = createMemoryStore(session);
+
+// store is used to store the session
+const store = new MemoryStore({
+  checkPeriod: 86400000, // prune expired entries every 24h
+});
+
+// Keeps an in-memory mapping between a SCIPER (userid) and its opened session
+// IDs. Needed to invalidate the sessions of a used when its role changes. The
+// value is a bag of sessions IDs.
+const sess2sciper = new Map<number, Map<string, undefined>>();
```

Comment:
What is the undefined field of the inner map ?

Reply:
The map is used as a bag/set. Javascript doesn't have a native data structure for that, so I am using a map where only the keys are used.

Reply:
Oh ok I see, although sets do exist in Javascript which should work for the session IDs, at least if it doesn't matter if duplicate session IDs for a user only appear once in the set ?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
Thanks for pointing that, I overlooked this native `Set` data-structure, which exactly fits the purpose !